ghost/members-api/lib/repositories/MemberRepository.js

```diff
@@ -1635,4 +1639,162 @@ module.exports = class MemberRepository {
         }
         return true;
     }
+
+    /**
+     * Force a Stripe customer to be subscribed to a specific Ghost product
+     *
+     * This will either:
+     *
+     * Create a new price on the Stripe product that is associated with the Ghost product, and update
+     * the customer's Stripe subscription to use the new price. The new price will be created with the details of the
+     * existing price of the item in customer's Stripe subscription
+     *
+     * or
+     *
+     * Update the customer's stripe subscription to use an existing price on the Stripe product that matches the
+     * details of the existing price of the item in customer's Stripe subscription
+     *
+     * If there is no Stripe product associated with the Ghost product, one will be created
+     *
+     * This method should be used in-conjunction with the `linkSubscription` method to ensure that the changes
+     * made in Stripe are reflected in Ghost - This is not executed as part of this to allow for flexibility
+     *
+     * @param {Object} data
+     * @param {String} data.customer_id - Stripe customer ID
+     * @param {String} data.product_id - Ghost product ID
+     * @param {Object} options
+     * @returns {Promise<Object>}
+     */
+    async forceStripeSubscriptionToProduct(data, options) {
```

Comment:
I don't think this method belongs in here - we want to decouple the `members-api` package from Stripe completely, and this takes us in the other direction.

I would consider moving this into the import package, or a separate one - but I think importer is fine for now

Reply:
Ok cool , I'll look at moving this into the `members-importer` package seeming though that this logic will probably only ever be called within the context of importing members 👍

Reply:
I've moved this logic over to the `members-importer` package now - ended up being a little trickier than expected to get the relevant dependencies passed around so had to rejig the package a little